python/pyarrow/tests/test_orc.py

```diff
@@ -110,20 +114,20 @@ def check_example_file(orc_path, expected_df, need_fix=False):
     json_pos = 0
     for i in range(orc_file.nstripes):
         batch = orc_file.read_stripe(i)
-        check_example_values(pd.DataFrame(batch.to_pydict()),
-                             expected_df,
-                             start=json_pos,
-                             stop=json_pos + len(batch))
+        check_example_values(
+            pd.DataFrame(batch.to_pydict()),
+            expected_df,
+            start=json_pos,
+            stop=json_pos + len(batch),
+        )
```

Comment:
Please avoid doing unrelated style changes.
Also, if you want an automated style pass, you can use `archery lint --python --fix`
(see https://arrow.apache.org/docs/developers/archery.html)